mlir/test/Conversion/ConvertToSPIRV/arith.mlir

```diff
@@ -1,4 +1,4 @@
-// RUN: mlir-opt -convert-to-spirv -split-input-file %s | FileCheck %s
+// RUN: mlir-opt -convert-to-spirv="run-signature-conversion=false" -split-input-file %s | FileCheck %s
```

Comment:
You could also make this a test pass so that the 'public' pass doesn't have these options meant for testing

Reply:
Added to "Future Plans".